project-chip/connectedhomeip
# AclStorage dependency injection

#### Problem
#14443

#### Change overview
Make AclStorage into an inteface and inject it into Server via initParams.

#### Testing
- Build and run (chip-all-clusters-app and chip-tool) on Linux x64.
- Verify that 2 fabrics with 2 ACL entries each survive restarting chip-all-clusters-app.
